.gitignore

```diff
@@ -172,3 +172,6 @@ UpgradeLog*.htm
 
 # Microsoft Fakes
 FakesAssemblies/
+
+# vendor dir for packages from nuget
+vendor/*
```

Comment:
If we make this `vendor/*/*` it should achieve the same result but not mask some of the files we already have checked in under vendor.